milvus_model/dense/jinaai.py

```diff
@@ -43,17 +44,24 @@ def dim(self):
         return self._dim
 
     def encode_queries(self, queries: List[str]) -> List[np.array]:
-        return self._call_jina_api(queries)
+        return self._call_jina_api(queries, task_type='retrieval.query')
 
     def encode_documents(self, documents: List[str]) -> List[np.array]:
-        return self._call_jina_api(documents)
+        return self._call_jina_api(documents, task_type='retrieval.passage')
 
-    def __call__(self, texts: List[str]) -> List[np.array]:
-        return self._call_jina_api(texts)
+    def __call__(self, texts: List[str], task_type: Optional[str] = None) -> List[np.array]:
+        return self._call_jina_api(texts, task_type=task_type)
 
-    def _call_jina_api(self, texts: List[str]):
+    def _call_jina_api(self, texts: List[str], task_type: Optional[str] = None):
+        data = {
+            "input": texts,
+            "model": self.model_name,
+            "dimensions": self.dimensions,
```

Comment:
please check that `self.dimensions` and `task_type` are not None before putting them in the `data`